lib/styleDictionary/util.js

```diff
@@ -77,8 +77,50 @@ const createTreeOfLeastDepth = (inputTokens, identifiers, tokenConverter) => {
   }, {});
 };
 
+/**
+ * Extracts a font family's base name, e.g. 'Roboto-Regular' => 'Roboto'
+ */
+const getBaseNameFromFontFamily = family => family.split('-')[0];
```

Comment:
What if the font is RobotoRegular.ttf?

Reply:
convention :)

Reply:
removed this functionality, since it's only relevant to web, will move it to the rendering target mapping